variables.tf

```diff
@@ -1,9 +1,90 @@
-########################################################################################################################
+##############################################################################
 # Input Variables
-########################################################################################################################
+##############################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to use"
+  type        = string
+  sensitive   = true
+  default     = null
+}
+
+variable "region" {
+  description = "Cluster region"
+  type        = string
+  nullable    = false
+}
+
+variable "cluster_id" {
+  type        = string
+  description = "Id of the target IBM Cloud OpenShift Cluster"
+  nullable    = false
+}
+
+variable "operator_helm_release_namespace" {
+  type        = string
+  description = "Namespace to deploy the helm releases. Default to liberty-operator-helm-release"
+  default     = "liberty-operator-helm-release"
```

Comment:
Is `liberty-operator` better than `liberty-operator-helm-release` for a namespace name?

Reply:
it is the namespace where to store the helm release, but I can reduce it, it is just the default value